apps/docs-snippets/scripts/pretest.sh

```diff
@@ -1,6 +1,6 @@
 # Check if node is already running at port 4000, if not start it
 # TODO: This is a temporary solution to avoid conflicts with the test node in docs-snippets2
-if ! lsof -t -i:4000 > /dev/null; then
+if ! lsof -t -i:4001 > /dev/null; then
```

Comment:
The changes I did in my PR on the [pretest and fuels.config.ts files](https://github.com/FuelLabs/fuels-ts/pull/3366/files#diff-5296244d79271c1537460f29b960617bfde246f5a676df241edcf3e61ae6ffe8R3) are on `docs-snippets2`. Could these changes to `docs-snippets` negate the changes on `docs-snippets2`? Perhaps we can revert these files. @petertonysmith94 What do you think?

Reply:
It would negate the changes, I'm happy just reverting the changes to this branch:
- [f53b56d](https://github.com/FuelLabs/fuels-ts/pull/3367/commits/f53b56df02d7e10a61f6134320fb80476bf3aaf6)
- [7a79002](https://github.com/FuelLabs/fuels-ts/pull/3367/commits/7a79002e3de7408312e9e58a9681ed72a864b521)

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3367/commits/9c7f5bd4c0272fe486e1bc2890870be2bb813bd0